packages/daemon/src/directory.js

```diff
@@ -84,6 +87,17 @@ export const makeDirectoryMaker = ({
       return hub.identify(name);
     };
 
+    /** @type {import('./types.js').EndoDirectory['locate']} */
+    const locate = async (...petNamePath) => {
+      const id = await identify(...petNamePath);
+      if (id === undefined) {
+        return undefined;
+      }
+
+      const formulaType = await getTypeForId(id);
+      return formatLocator(id, formulaType);
```

Comment:
Wherein the protagonist verifies that the L in URL is for Locator as opposed to Location.